core/kazoo_yodb/src/kazoo_yodb.app.src

```diff
@@ -0,0 +1,13 @@
+{application, kazoo_yodb,
+ [
+  {description, "YODb - Utilities for an account's year-only database"},
+  {vsn, "4.0.0"},
+  {modules, []},
+  {registered, [ ]},
+  {applications, [ kernel
```

Comment:
You might try running `make apps_of_app` in the yodb directory to properly populate this file.